### ISSUE TITLE
Comments

### ISSUE THREAD

**Issue by AyaSF** [SeleniumHQ/selenium-ide#245]
Hi,

I miss the old comment line: actually it is not possible to add comment without command.
Could you add the improvement?

With this improvment, could you show the comment lines in different color?
And could you import the comments of test cases of old version of Selenium?

Thank you by advance for all of these things!
AyaSF

**Comment by corevo** [comment]
Regarding the first point, in the next version this is already fixed, you can have a comment with an empty command.  
Regarding the second point, in my opinion comments already have their unique icon left of the command table, maybe adding a color is a possibility.  
Regarding the last point is a duplicate of #237 which you opened.  
Also as a side note, please keep to the issue template, it helps us manage a large amounts of issues more easily.

**Comment by corevo** [comment]
Regarding the first point, I said the **next version**, so next time Selenium IDE will be updated you'll have that.  
The second point, is a matter of taste, I'm gonna ask around see what people in the office think.  
The third point, then as I said in #237 attach a test case with this issue, you can censor any information in it, also open its own issue, it is hard to follow a thread with 3 different issues in it, it requires maintainers to read the whole thread to understand the context.

**Comment by AyaSF** [comment]
For the first point, sorry it is a misunderstanding.

For the second point, could you let this issue open at least for this subject?

For the third point, i thought visibility and color of comments are the same theme.
And error on import with special cases an other.
But like you want, just hope the fixes ;)

Thank you by advance for all.
AyaSF
